keyboards/hardwareabstraction/handwire/info.json

```diff
@@ -0,0 +1,102 @@
+{
+    "manufacturer": "Hardware Abstraction",
+    "keyboard_name": "handwire",
+    "maintainer": "atsai",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+
+    "matrix_pins": {
+        "cols": ["F0", "F1", "F4", "F5", "F6", "F7", "C7", "C6", "B6", "B5", "B4", "D7", "D6", "D4"],
+        "rows": ["B0", "B7", "D5", "D3", "D2"]
+    },
+
+    "processor": "atmega32u4",
+    "url": "",
```

Comment:
```suggestion
    "url": "https://github.com/atsai",
```

Reply:
Changed with commit [f8317e8](https://github.com/qmk/qmk_firmware/pull/18504/commits/f8317e85fba33d1f63c420666689ee9642765db6)

Reply:
Is it supposed to be `atsai` or `Cgmandy`?
The former has nothing of note, especially nothing QMK-related.

Reply:
It should be `Cgmandy`. I think drashna assumed the URL would be githu.com/atsai because originally in the info.json, the maintainer was marked 'atsai' (my bad). In commit [f8317e8](https://github.com/qmk/qmk_firmware/pull/18504/commits/f8317e85fba33d1f63c420666689ee9642765db6), both the maintainer field and the url field have been changed to the correct usernames.